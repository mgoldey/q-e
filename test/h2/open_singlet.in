&CONTROL
  calculation  = "scf",
  prefix       = "left",
  outdir       = "./out",
  pseudo_dir   = "../pseudo",
  do_epcdft    = .true.,
  verbosity = "high" 
  wf_collect = .true.
/
&SYSTEM
  nosym = .true.
  ibrav     = 1,
  a         = 12,
  nat       = 2,
  ntyp      = 2,
  ecutwfc   = 60,
  nspin   = 2,
  tot_charge = 0
  tot_magnetization = 0
  starting_magnetization(1)=1
  starting_magnetization(2)=-1

/
&ELECTRONS
 ! startingpot = 'file'
 ! startingwfc = 'file'
  conv_thr    = 1.D-8,
  mixing_beta = 0.30,
/

EPCDFT 
1 1e-2 1e-2 
delta_spin  1 1 2 2 2.0 0.2
!delta_charge 1 1 2 2 2.0 0.9

ATOMIC_SPECIES
H1 0.0 H.pbe-mt_fhi.UPF
H2 0.0 H.pbe-mt_fhi.UPF

ATOMIC_POSITIONS (angstrom)
H1 0 0 0
H2 5 0 0

K_POINTS {Gamma}
