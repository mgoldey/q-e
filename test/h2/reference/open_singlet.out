
     Program PWSCF v.5.1.1 starts on 12Feb2016 at 15:27:32 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote

     Parallel version (MPI & OpenMP), running on       8 processor cores
     Number of MPI processes:                 1
     Threads/MPI process:                     8
     Reading input from open_singlet.in
 Reading EPCDFT CARD. Printing based upon belief that input should be reproducab
 le from output.
     Reading   1 constraint(s); tolerance:    0.010000 max change in field 0.01
          constraint           type   A   A   D   D      T      L
                   1  delta_spin      1   1   2   2 2.0000 0.2000

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
               file H.pbe-mt_fhi.UPF: wavefunction(s)  2p 4f renormalized
               file H.pbe-mt_fhi.UPF: wavefunction(s)  2p 4f renormalized

     gamma-point specific algorithms are used

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     G-vector sticks info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Sum        9809    9809   2449               732287   732287   91623
     Tot        4905    4905   1225
 
     Generating pointlists ...
     new r_m :   0.1719 (alat units)  3.8976 (a.u.) for type    1
     new r_m :   0.1719 (alat units)  3.8976 (a.u.) for type    2


     bravais-lattice index     =            1
     lattice parameter (alat)  =      22.6767  a.u.
     unit-cell volume          =   11661.1222 (a.u.)^3
     number of atoms/cell      =            2
     number of atomic types    =            2
     number of electrons       =         2.00 (up:   1.00, down:   1.00)
     number of Kohn-Sham states=            4
     kinetic-energy cutoff     =      60.0000  Ry
     charge density cutoff     =     240.0000  Ry
     convergence threshold     =      1.0E-06
     mixing beta               =       0.3000
     number of iterations used =            8  plain     mixing
     Exchange-correlation      = PBE ( 1  4  3  4 0 0)

     celldm(1)=  22.676714  celldm(2)=   0.000000  celldm(3)=   0.000000
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   1.000000 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  1.000000 )  


     PseudoPot. # 1 for  H read from file:
     ../pseudo/H.pbe-mt_fhi.UPF
     MD5 check sum: a19bd5488733b3d98581df861506e80d
     Pseudo is Norm-conserving, Zval =  1.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.2
     Using radial grid of  387 points,  3 beta functions with: 
                l(1) =   0
                l(2) =   1
                l(3) =   3

     PseudoPot. # 2 for  H read from file:
     ../pseudo/H.pbe-mt_fhi.UPF
     MD5 check sum: a19bd5488733b3d98581df861506e80d
     Pseudo is Norm-conserving, Zval =  1.0
     Generated using FHI98PP, converted with fhi2upf.x v.5.0.2
     Using radial grid of  387 points,  3 beta functions with: 
                l(1) =   0
                l(2) =   1
                l(3) =   3

     atomic species   valence    mass     pseudopotential
        H1             1.00     1.00794      H( 1.00)
        H2             1.00     1.00794      H( 1.00)

     Starting magnetic structure 
     atomic species   magnetization
        H1           1.000
        H2          -1.000

     No symmetry found


                                    s                        frac. trans.

      isym =  1     identity                                     

 cryst.   s( 1) = (     1          0          0      )
                  (     0          1          0      )
                  (     0          0          1      )

 cart.    s( 1) = (  1.0000000  0.0000000  0.0000000 )
                  (  0.0000000  1.0000000  0.0000000 )
                  (  0.0000000  0.0000000  1.0000000 )


     point group C_1 (1)    
     there are  1 classes
     the character table:

       E    
A      1.00

     the symmetry operations in each class:
     E        1

   Cartesian axes

     site n.     atom                  positions (alat units)
         1           H1  tau(   1) = (   0.0000000   0.0000000   0.0000000  )
         2           H2  tau(   2) = (   0.4166667   0.0000000   0.0000000  )

   Crystallographic axes

     site n.     atom                  positions (cryst. coord.)
         1           H1  tau(   1) = (  0.0000000  0.0000000  0.0000000  )
         2           H2  tau(   2) = (  0.4166667  0.0000000  0.0000000  )

     number of k points=     2
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

                       cryst. coord.
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000
        k(    2) = (   0.0000000   0.0000000   0.0000000), wk =   1.0000000

     Dense  grid:   366144 G-vectors     FFT dimensions: ( 120, 120, 120)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         2.80 Mb     (   45812,    4)
        NL pseudopotentials            15.38 Mb     (   45812,   22)
        Each V/rho on FFT grid         52.73 Mb     ( 1728000,   2)
        Each G-vector array             2.79 Mb     (  366144)
        G-vector shells                 0.02 Mb     (    2609)
     Largest temporary arrays     est. size (Mb)     dimensions
        Each subspace H/S matrix        0.00 Mb     (       4,    4)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      22,    4)
        Arrays for rho mixing         210.94 Mb     ( 1728000,    8)

     Initial potential from superposition of free atoms

     starting charge    1.99998, renormalised to    2.00000

     negative rho (up, down):  6.014E-05 6.014E-05
     Starting wfc are   32 randomized atomic wfcs

     total cpu time spent up to now is        7.8 secs

     per-process dynamical memory:   479.6 Mb

     Self-consistent Calculation

     iteration #  1     ecut=    60.00 Ry     beta=0.30
     CG style diagonalization
     ethr =  1.00E-02,  avg # of iterations =  3.5

     Threshold (ethr) on eigenvalues was too large:
     Diagonalizing with lowered threshold

     CG style diagonalization
     ethr =  5.94E-05,  avg # of iterations =  9.0

     negative rho (up, down):  4.166E-05 4.165E-05
 
     Magnetic moment per site:
     atom:    1    charge:    0.9819    magn:    0.9817    constr:    0.0000
     atom:    2    charge:    0.9819    magn:   -0.9817    constr:    0.0000

     total cpu time spent up to now is       15.7 secs

     total energy              =      -2.01766271 Ry
     Harris-Foulkes estimate   =      -2.01894109 Ry
     estimated scf accuracy    <       0.00343505 Ry

     total magnetization       =    -0.00 Bohr mag/cell
     absolute magnetization    =     2.00 Bohr mag/cell

     iteration #  2     ecut=    60.00 Ry     beta=0.30
     CG style diagonalization
     ethr =  1.72E-04,  avg # of iterations =  3.0

     negative rho (up, down):  3.866E-06 3.866E-06
 
     Magnetic moment per site:
     atom:    1    charge:    0.9867    magn:    0.9866    constr:    0.0000
     atom:    2    charge:    0.9867    magn:   -0.9866    constr:    0.0000

     total cpu time spent up to now is       18.8 secs

     total energy              =      -2.01144878 Ry
     Harris-Foulkes estimate   =      -2.01811584 Ry
     estimated scf accuracy    <       0.00150693 Ry

     total magnetization       =     0.00 Bohr mag/cell
     absolute magnetization    =     2.00 Bohr mag/cell

     iteration #  3     ecut=    60.00 Ry     beta=0.30
     CG style diagonalization
     ethr =  7.53E-05,  avg # of iterations =  3.0

     negative rho (up, down):  3.945E-06 3.943E-06
 
     Magnetic moment per site:
     atom:    1    charge:    0.9866    magn:    0.9865    constr:    0.0000
     atom:    2    charge:    0.9866    magn:   -0.9865    constr:    0.0000

     total cpu time spent up to now is       21.9 secs

     total energy              =      -2.01164912 Ry
     Harris-Foulkes estimate   =      -2.01163493 Ry
     estimated scf accuracy    <       0.00000891 Ry

     total magnetization       =    -0.00 Bohr mag/cell
     absolute magnetization    =     2.00 Bohr mag/cell

     iteration #  4     ecut=    60.00 Ry     beta=0.30
     CG style diagonalization
     ethr =  4.45E-07,  avg # of iterations =  5.0

     negative rho (up, down):  3.949E-06 3.956E-06
 
     Magnetic moment per site:
     atom:    1    charge:    0.9864    magn:    0.9863    constr:    0.0000
     atom:    2    charge:    0.9865    magn:   -0.9864    constr:    0.0000

     total cpu time spent up to now is       25.9 secs

     total energy              =      -2.01344407 Ry
     Harris-Foulkes estimate   =      -2.01168448 Ry
     estimated scf accuracy    <       0.00000825 Ry

     total magnetization       =     0.00 Bohr mag/cell
     absolute magnetization    =     2.00 Bohr mag/cell

     iteration #  5     ecut=    60.00 Ry     beta=0.30
     CG style diagonalization
     ethr =  4.12E-07,  avg # of iterations =  3.1

     negative rho (up, down):  2.825E-06 2.844E-06
 
     Magnetic moment per site:
     atom:    1    charge:    0.9863    magn:    0.9862    constr:    0.0000
     atom:    2    charge:    0.9864    magn:   -0.9863    constr:    0.0000

     total cpu time spent up to now is       29.2 secs

     total energy              =      -2.01474092 Ry
     Harris-Foulkes estimate   =      -2.01347117 Ry
     estimated scf accuracy    <       0.00000158 Ry

     total magnetization       =    -0.00 Bohr mag/cell
     absolute magnetization    =     2.00 Bohr mag/cell

     iteration #  6     ecut=    60.00 Ry     beta=0.30
     CG style diagonalization
     ethr =  7.88E-08,  avg # of iterations =  3.4
 
     Magnetic moment per site:
     atom:    1    charge:    0.9865    magn:    0.9864    constr:    0.0000
     atom:    2    charge:    0.9865    magn:   -0.9865    constr:    0.0000

     total cpu time spent up to now is       32.2 secs

     End of self-consistent calculation

 ------ SPIN UP ------------


          k = 0.0000 0.0000 0.0000 ( 45812 PWs)   bands (ev):

   -10.3797  -1.5850  -1.4590  -1.4589

     occupation numbers 
     1.0000   0.0000   0.0000   0.0000

 ------ SPIN DOWN ----------


          k = 0.0000 0.0000 0.0000 ( 45812 PWs)   bands (ev):

   -10.3831  -1.5819  -1.4562  -1.4560

     occupation numbers 
     1.0000   0.0000   0.0000   0.0000

     highest occupied, lowest unoccupied level (ev):   -10.3797   -1.5850
       I     D       A      Val   Target      Str
       1 0.999   0.999    1.998    2.000    0.200

!    total energy              =      -1.99635339 Ry
     Harris-Foulkes estimate   =      -2.01476941 Ry
     estimated scf accuracy    <       0.00000080 Ry

     The total energy is the sum of the following terms:

     one-electron contribution =      -1.50460052 Ry
     hartree contribution      =       0.98485656 Ry
     xc contribution           =      -1.22868173 Ry
     ewald contribution        =      -0.24792770 Ry
     CDFT correction           =       0.39953751 Ry

     total magnetization       =    -0.00 Bohr mag/cell
     absolute magnetization    =     2.00 Bohr mag/cell

     convergence has been achieved in   6 iterations

     Writing output data file left.save
 
     init_run     :     43.18s CPU      7.77s WALL (       1 calls)
     electrons    :    187.35s CPU     25.05s WALL (       1 calls)

     Called by init_run:
     wfcinit      :     14.79s CPU      2.02s WALL (       1 calls)
     potinit      :     26.09s CPU      4.54s WALL (       1 calls)

     Called by electrons:
     c_bands      :     89.30s CPU     11.78s WALL (       7 calls)
     sum_band     :     17.04s CPU      2.27s WALL (       7 calls)
     v_of_rho     :     66.14s CPU      9.32s WALL (       7 calls)
     v_h          :      3.46s CPU      0.47s WALL (       7 calls)
     v_xc         :     62.68s CPU      8.85s WALL (       7 calls)
     mix_rho      :     10.53s CPU      1.43s WALL (       7 calls)

     Called by c_bands:
     init_us_2    :      4.82s CPU      0.71s WALL (      30 calls)
     rcgdiagg     :     79.08s CPU     10.41s WALL (      14 calls)
     wfcrot       :     19.52s CPU      2.57s WALL (      12 calls)

     Called by sum_band:

     Called by *cgdiagg:
     h_psi        :     91.45s CPU     12.03s WALL (     212 calls)
     rdiaghg      :      0.00s CPU      0.00s WALL (      12 calls)

     Called by h_psi:
     h_psi:vloc   :     83.48s CPU     10.98s WALL (     212 calls)
     h_psi:vnl    :      7.49s CPU      0.98s WALL (     212 calls)
     add_vuspsi   :      3.16s CPU      0.42s WALL (     212 calls)

     General routines
     calbec       :      7.55s CPU      0.98s WALL (     412 calls)
     fft          :     46.45s CPU      6.21s WALL (     164 calls)
     fftw         :     64.81s CPU      8.47s WALL (     532 calls)
 
     Parallel routines
     fft_scatter  :     37.10s CPU      4.96s WALL (     696 calls)
 
     PWSCF        :  3m52.42s CPU     0m33.25s WALL

 
   This run was terminated on:  15:28: 5  12Feb2016            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
